kbase-extension/static/kbase/js/common/cellComponents/fieldTableCellWidget.js

```diff
@@ -63,15 +62,6 @@ define([
                             class: `${cssBaseClass}__cell_label`,
                             title: spec.ui.label || spec.ui.id,
                             for: ids.inputControl,
-                            id: events.addEvent({
```

Comment:
Doesn't appear to be used anywhere, so I just removed it and other references to the unused 'events'

Reply:
Since the element it references doesn't even exist, I don't think it'll be any great loss!